pkg/systemd/generate/common.go

```diff
@@ -53,3 +55,18 @@ func filterPodFlags(command []string) []string {
 	}
 	return processed
 }
+
+// regexWhitespace matches string that contain at least one whitespace.
+var regexWhitespace = regexp.MustCompile(`^.*\s.*$`)
+
+// quoteArguments makes sure that all arguments with at least one whitespace
+// are quoted to make sure those are interpreted as one argument instead of
+// multiple ones.
+func quoteArguments(command []string) []string {
+	for i := range command {
+		if regexWhitespace.MatchString(command[i]) {
```

Comment:
regex seems like such overkill. Did you consider `string.ContainsAny` with `"<space><tab>"`?

Reply:
Good idea, I'll change!